<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>curvature</class>
 <widget class="QMainWindow" name="curvature">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="curCurvature">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Curvature:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="curAngle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Angle:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="curPixellength">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Length(pixel):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Length(mm):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="scaleCalibration">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>10</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="paintable" name="draw">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSpinBox" name="photoIndex">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="photoList">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="lineAddbutton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>New Item</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearMeasurebutton">
          <property name="text">
           <string>New Measure</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rowAddbutton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Add Data</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actiondataexp"/>
   </widget>
   <widget class="QMenu" name="menuMeasure">
    <property name="title">
     <string>Measure</string>
    </property>
    <addaction name="actiongrid"/>
    <addaction name="actionscale"/>
    <addaction name="actioncurvature"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionRead_me"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMeasure"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionscale">
   <property name="text">
    <string>Scale</string>
   </property>
  </action>
  <action name="actioncurvature">
   <property name="text">
    <string>Curvature</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actiondataexp">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actiongrid">
   <property name="text">
    <string>Grid</string>
   </property>
  </action>
  <action name="actionsave_picture">
   <property name="text">
    <string>save picture</string>
   </property>
  </action>
  <action name="actionRead_me">
   <property name="text">
    <string>Read me</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>paintable</class>
   <extends>QLabel</extends>
   <header location="global">paintable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
